src/libraries/Microsoft.PowerFx.Core/Public/Types/TableType.cs

```diff
@@ -35,9 +36,12 @@ public TableType Add(NamedFormulaType field)
             return new TableType(newType);
         }
 
+        public string SingleColumnFieldName => GetNames().First().Name;
+        public FormulaType SingleColumnFieldType => GetNames().First().Type;
+
```

Comment:
We may want to assert (or throw an exception) if this is not a single-column table

Reply:
Done